test/setup/iam.tf

```diff
@@ -15,13 +15,26 @@
  */
 
 locals {
-  int_required_roles = [
+  per_module_roles = {
+    simple-secret = [
+      "roles/secretmanager.secretAccessor",
```

Comment:
As mentioned in the services block, shouldn't this also include,

* roles/cloudkms.admin
* roles/pubsub.admin

Also should we use `roles/secretmanager.admin` for secret manager roles as used before this change?